api/app/signals/apps/signals/migrations/0142_auto_20210727_1406.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.5 on 2021-07-27 12:06
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('signals', '0141_workflow_additions'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='routingexpression',
+            name='_user',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL),
+        ),
+        migrations.AlterField(
+            model_name='status',
+            name='state',
+            field=models.CharField(blank=True, choices=[('m', 'Gemeld'), ('i', 'In afwachting van behandeling'), ('b', 'In behandeling'), ('h', 'On hold'), ('ingepland', 'Ingepland'), ('ready to send', 'Extern: te verzenden'), ('o', 'Afgehandeld'), ('a', 'Geannuleerd'), ('reopened', 'Heropend'), ('s', 'Gesplitst'), ('closure requested', 'Extern: verzoek tot afhandeling'), ('reaction requested', 'Reactie gevraagd'), ('reaction received', 'Reactie ontvangen'), ('sent', 'Extern: verzonden'), ('send failed', 'Extern: mislukt'), ('done external', 'Extern: afgehandeld'), ('reopen requested', 'Verzoek tot heropenen')], default='m', help_text='Melding status', max_length=20),
+        ),
+        migrations.AlterField(
```

Comment:
Good find! 👍🏼  We missed this, perhaps we can fix this in a separate PR so that this PR is only about user assignment.